packages/admin-ui/src/AutoComplete/primitives/AutoCompletePrimitive.tsx

```diff
@@ -116,22 +120,20 @@ const AutoCompletePrimitive = (props: AutoCompletePrimitiveProps) => {
                         />
                     </span>
                 </Popover.Trigger>
-                <Popover.Portal>
-                    <Popover.Content
-                        style={{ width: "var(--radix-popover-trigger-width)" }}
-                        onOpenAutoFocus={e => e.preventDefault()}
-                    >
-                        <AutoCompleteList
-                            options={vm.optionsListVm.options}
-                            onOptionSelect={handleSelectOption}
-                            isEmpty={vm.optionsListVm.isEmpty}
-                            isLoading={props.isLoading}
-                            loadingMessage={vm.optionsListVm.loadingMessage}
-                            emptyMessage={vm.optionsListVm.emptyMessage}
-                            optionRenderer={props.optionRenderer}
-                        />
-                    </Popover.Content>
-                </Popover.Portal>
+                <Popover.Content
```

Comment:
It says here popover was removed:
![image](https://github.com/user-attachments/assets/b8abccb2-3ff0-4579-881b-b34b3fe06e4d)

But here it's actually here.

What did I get wrong? 😇

Reply:
I've checked; it has been removed from the Autocomplete component because it's already placed inside `Popover.Content`.